### DISCUSSION TITLE
View list of active sessions

### DISCUSSION THREAD

**Discussion by citroid** [OpenIDC/mod_auth_openidc#1231] [Q&A] [answered]
Hi,
I wonder if there is a way to view a list of all currently active sessions in mod_auth_openidc?
Kind regards,
citroid

**Comment by zandbelt** [comment, marked as answer]
there's no "API' for that, but one could inspect the (session) cache contents, which typically works best with `OIDCCacheType file`, but looping over a Memcache or Redis server may also be possible, especially if the contents are not encrypted

**Reply by citroid** [reply]
Thank you for the quick response!
I'm using the default option "shared memory" for OIDCCacheType, which makes it difficult to inspect this data. 
So I changed the settings in my testing environment to cache type "file", turned encryption off and was able to find the session data in a default directory in /tmp.
